feature-libs/cart/saved-cart/components/page-context/saved-carts-page/new-saved-cart-order-entries.context.ts

```diff
@@ -39,6 +41,23 @@ export class NewSavedCartOrderEntriesContext implements AddOrderEntriesContext {
     savedCartInfo?: { name: string; description: string }
   ): Observable<ProductImportInfo> {
     return this.add(products, savedCartInfo).pipe(
+      tap((cartId: string) => {
+        this.importInfoService
+          .getResults(cartId)
+          .pipe(
+            take(products.length),
+            every(
+              (productInfo: ProductImportInfo) =>
+                productInfo.statusCode ===
+                ProductImportStatus.UNKNOWN_IDENTIFIER
```

Comment:
I think we should also look for UNKNOWN_ERROR

Reply:
Alright, just pushed it, let me know if any change should be made to the tests :)